### PR TITLE
Add EnviroMc to the shared hosts list.

### PR DESCRIPTION
**Pull request by KoxSosen**
I added [EnviroMc](https://enviromc.host/index.php) to the hosts list.

### CODE REVIEW

**Comment by KoxSosen** [comment]
image

**Review by Proximyst — CHANGES_REQUESTED** [review]
Host must be put in alphabetical order to ensure we remain impartial.

**Comment by KoxSosen** [comment]
Done ^_^

File: source/java-update/index.rst [hunk 1]
```diff
@@ -88,6 +88,17 @@ For more information, feel free to read the `How to change your server's Java
 version <https://docs.dedicatedmc.io/server-setup/how-to-change-your-servers-java-version/>`_
 guide on their wiki.
 
+EnviroMC
+~~~~~~~~
+
+EnviroMC defaults to Java 16 if you select Paper as your server jar.
+To manually change your Java version, please navigate to the
+Startup -> Docker Image option, and select Java 16, as shown below.
```
Comment (Proximyst) [inline source/java-update/index.rst:96]: Please italicise the website locations as done for other hosts.

**Review by Proximyst — APPROVED** [review]
Otherwise, looks fine to me

File: source/java-update/index.rst (hunk 1 above)
Comment (KoxSosen) [reply source/java-update/index.rst]: Done, please let me know if this is good.

**Pull request merged**